akka-actor-tests/src/test/scala/akka/actor/ReceiveTimeoutSpec.scala

```diff
@@ -102,7 +102,12 @@ class ReceiveTimeoutSpec extends AkkaSpec {
         }
       }))
 
-      val ticks = system.scheduler.schedule(100.millis, 100.millis, timeoutActor, TransperentTick)(system.dispatcher)
+      val ticks = system.scheduler.schedule(100.millis, 100.millis, new Runnable {
+        override def run() = {
+          timeoutActor ! TransperentTick
+          timeoutActor ! Identify(None)
+        }
+      })(system.dispatcher)
```

Comment:
I don't think this test change is needed, as it already tests that something marked with `NotInfluenceReceiveTimeout` does not influence the timeout, and that logic has not changed.

Essentially the needed test would be that `Identify` is a `NotInfluenceReceiveTimeout` which may not really be a reasonable test.